app/controllers/single_reservations_controller.rb

```diff
@@ -0,0 +1,84 @@
+class SingleReservationsController < ApplicationController
+
+  customer_tab  :all
+  before_action :authenticate_user!
+  load_resource :facility, find_by: :url_name
+  load_resource :instrument, through: :facility, find_by: :url_name
+  before_action :build_order
+  before_action { @submit_action = facility_instrument_single_reservations_path }
```

Comment:
This seemed to be the simplest way of still using the same view partial as the old reservations form. I'm open to other ideas.